doc/atcommands.txt

```diff
@@ -1420,7 +1420,7 @@ Affected files:
 -- pcdb: statpoint.txt, job_exp.txt, skill_tree.txt, attr_fix.txt, job_db1.txt, job_db2.txt, job_basehpsp_db.txt, job_param_db.txt, level_penalty.yml
 -- questdb: quest_db.yml
 -- script: /npc/*.txt, /npc/*.conf
--- skilldb: skill_db.yml, const.txt, skill_nocast_db.txt, skill_changematerial_db.txt, skill_damage_db.txt, abra_db.yml, create_arrow_db.txt, produce_db.txt, spellbook_db.yml, magicmushroom_db.yml
+-- skilldb: skill_db.yml, const.yml, skill_nocast_db.txt, skill_changematerial_db.txt, skill_damage_db.txt, abra_db.yml, create_arrow_db.txt, produce_db.txt, spellbook_db.yml, magicmushroom_db.yml
```

Comment:
Why does reloadskilldb trigger a script constant reload...? x_x

Reply:
It seems the doc was wrong anyway
```
	} else if (strstr(command, "skilldb") || strncmp(message, "skilldb", 4) == 0) {
		skill_reload();
		hom_reload_skill();
		reload_elemental_skilldb();
		mercenary_read_skilldb();
		clif_displaymessage(fd, msg_txt(sd,99)); // Skill database has been reloaded.
```